script/requirements.txt

```diff
@@ -1,8 +1,70 @@
-# This requirements.txt file is used to install the necessary packages to run the python scripts in
-# this scripts folder
-jinja2==2.11.3
-PyYAML==5.4.1
-jsonschema==3.2.0
-pandas==1.1.4
-requests==2.22.0
-markupsafe==2.0.1
+aiohttp==3.7.4.post0; python_version >= "3.8" and python_version < "3.11"
```

Comment:
Why did this expand so much / why do we have both requirements.txt and poetry in here? 😅

Reply:
ah I just wanted to do this since it helps with enforcing a given Python version; I exported to requirements.txt to make the existing CI work although it's going to be made immediately redundant by https://github.com/cal-itp/data-infra/pull/2262